SafetyCulture/API-Documentation
# [EAG-259]

Update API document to include `item.options.failed_responses` and `item.responses.failed` into audit JSON data.

### Testing
* Set up the doc server as instructed in the readme of this repo
* Check the logic and wording of the added field
